### ISSUE TITLE
Use the Terminal to print out RPC response instead of printlns

### ISSUE THREAD

**Issue by etorreborre** [build-trust/ockam#5904]
[This PR](https://github.com/build-trust/ockam/pull/5901) refactored some commands which were using `println!` to print results to the console directly.

This makes it now possible to direct those messages to the `Terminal` present in the [`CommandGlobalOpts`](https://github.com/build-trust/ockam/blob/8db976d0433ceb960854382ce0ee53abc5039242/implementations/rust/ockam/ockam_command/src/lib.rs#L233) and benefit from a more controlled output (like being able to silence those messages if necessary).

[A TODO has been left in the code](https://github.com/build-trust/ockam/pull/5901/files#diff-f5a30aef85b96ee0c58c26d9c22690c484ea212154879d380cae8bd4b27ce018R250) to mark the place where modifications can take place.

Please also check if the use of `print!` in [this code](https://github.com/build-trust/ockam/pull/5901#discussion_r1318209384) should better be a `println!` instead.

---

<i>We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).</i>

**Comment by Tashuuuu** [comment]
There are many issues like #6491, #6490, #6489, #6488 and #6487 they specify which ```println``` should be replaced by ```Terminal```
So if there are more ```println``` that need to be replaced by ```Terminal```, please mention them. I would like to work on this issue

**Comment by nazmulidris** [comment]
@Tashuuuu I've assigned you to this issue 👍🏽 . Once you land a PR for this one, please take on another issue then 🙏🏽 .
